Engine/RealTime/ScheduledEventFactory.cs

```diff
@@ -90,6 +90,7 @@ public static ScheduledEvent EveryAlgorithmEndOfDay(IAlgorithm algorithm, IResul
                 try
                 {
                     algorithm.OnEndOfDay();
+                    algorithm.Debug("Usage of QCAlgorithm.OnEndOfDay() without a symbol will be deprecated August 2021. Always use a symbol when overriding this method: OnEndOfDay(symbol)");
```

Comment:
I think this should be logged **once** at the start, here we are logging it on each call?

Reply:
Gotcha, moved it to only log once when creating the event.